src/main/java/net/glowstone/block/itemtype/ItemWearable.java

```diff
@@ -0,0 +1,60 @@
+package net.glowstone.block.itemtype;
+
+import org.bukkit.Material;
+import org.bukkit.entity.LivingEntity;
+import org.bukkit.inventory.ItemStack; 
+
+public class ItemWearable extends ItemType {
```

Comment:
I still don't understand what ItemWearable is used for the dispenser logic.

If you're going to use it to equip items, just use `EnchantmentTarget.includes` for normal armor, and then a few extra boolean checks for the other non-enchantable materials, like pumpkins or skulls. 

You could make some sort of utility class for it too, to check for those extra materials, as well as to do the correct equipping for each equipment slot type.

I just don't think this data needs to be instanced per material in a new wearable class.